.github/workflows/github_actions.yml

```diff
@@ -55,7 +55,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        arch: [armv7, aarch64]
+        arch_with_features: [{arch: armv7, feature: none}, {arch: aarch64, feature: none}, {arch: aarch64, feature: crypto}]
```

Comment:
I have changed the subject of this pull request and would like to address `crc` feature in CI pipeline as well.

Reply:
Thanks! Btw should we remove the current crc enabling in Makefile line 33?

Reply:
Yes, go ahead.

Reply:
done